### ISSUE TITLE
Create company name detector

### ISSUE THREAD

**Issue by thomasbird** [LeapBeyond/scrubadub#63]
A detector to detect company names would be useful.

**Comment by aCampello** [comment]
This can be partially achieved by setting the Spacy NER detector to recognise `ORG` atm..

**Comment by thomasbird** [comment]
Yeah you can currently use `scrubadub_spacy.detectors.SpacyEntityDetector`, but I think we should investigate adding a little more finesse to it. Perhaps we can do something like the `scrubadub_spacy.detectors.SpacyNameDetector` that also looks for context, using words like company, ltd, limited, etc. I'm just leaving this comment as a reminder for the future.
